### PR TITLE
Fix conflict upstream

### PR DESCRIPTION
**Pull request by kgilmer**

### CODE REVIEW

**Comment by kgilmer** [comment]
related: https://github.com/regolith-linux/voulage/commit/d45c7934c39979c3ca5a9d0d8abb7400f8892dc5

**Comment by kgilmer** [comment]
see: https://github.com/regolith-linux/sway-regolith/pull/21

**Review by SoumyaRanjanPatnaik — COMMENTED** [review]
Question: Do we keep the base sway accessible? If yes, then we should probably leave the configuration via intact to ensure consistency in behaviour across the sessions.

File: debian/patches/01-regolith-trawl.patch [hunk 1]
```diff
@@ -49,25 +49,6 @@ Index: sway-regolith/meson.build
...
  	'libinput_backend': false,
-@@ -248,7 +269,7 @@ configure_file(
- )
- 
- install_data(
--	'sway.desktop',
-+	'sway-regolith.desktop',
- 	install_dir: join_paths(datadir, 'wayland-sessions')
- )
-
```
Comment (SoumyaRanjanPatnaik) [inline debian/patches/01-regolith-trawl.patch:60]: We should remove this section from meson.build if we don't want to keep the base session accessible. 

```patch
-  install_data( 
-           'sway.desktop',
-           'sway-regolith.desktop',
-           install_dir: join_paths(datadir, 'wayland-sessions')
- )
```

File: debian/patches/05-remove-config [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+-config = configuration_data()
+-config.set('datadir', join_paths(prefix, datadir))
+-config.set('prefix', prefix)
+-config.set('sysconfdir', join_paths(prefix, sysconfdir))
+-
+-configure_file(
+-	configuration: config,
+-	input: 'config.in',
+-	output: '@BASENAME@',
+-	install_dir: join_paths(sysconfdir, 'sway')
```
Comment (SoumyaRanjanPatnaik) [inline debian/patches/05-remove-config]: Removing these configuration files can render the base session non-functional. Either we remove the desktop files to make the session inaccessible, or make sure the configs are available as well.

**Review by SoumyaRanjanPatnaik — CHANGES_REQUESTED** [review]

File: debian/patches/01-regolith-trawl.patch (hunk 1 above)
Comment (kgilmer) [reply debian/patches/01-regolith-trawl.patch:60]: removed desktop session file as suggested, thanks

**Review by SoumyaRanjanPatnaik — APPROVED** [review]
LGTM

**Pull request merged**